Resources/Prototypes/Corvax/Entities/Clothing/OuterClothing/coats.yml

```
@@ -17,7 +17,7 @@
           Heat: 0.7
 
 - type: entity
-  parent: ClothingOuterStorageBase
+  parent: ClothingOuterCoatDetective
```

```suggestion
  parent: ClothingOuterCoatDetectiveLoadout
```
шобы не было зажигалочки гравированой

---

зажигалка с добавления лоадаутов спавнится в шкафу детектива, а у него в плаще - дефолтная белая
```yaml
- type: entity
  parent: ClothingOuterCoatDetective
  id: ClothingOuterCoatDetectiveLoadout
  components:
  - type: StorageFill
    contents:
    - id: SmokingPipeFilledTobacco
    - id: FlippoLighter #Not the steal objective, only difference from normal detective trenchcoat
```
армор у айтемов тоже **альтернативный**, в этом, типо, их смысл...
ClothingOuterCoatHoSTrench:
```yaml
      - type: Armor
    modifiers:
      coefficients:
        Blunt: 0.7
        Slash: 0.7
        Piercing: 0.7
        Heat: 0.7
        Caustic: 0.75 # not the full 90% from ss13 because of the head
```
 
 наша альтернатива:
 ```yaml
    - type: Armor
      modifiers:
        coefficients:
          Blunt: 0.7
          Slash: 0.7
          Piercing: 0.4
          Heat: 0.7
```
 про спрайт ты прав
 
 детективный плащ:
 ```yaml
   - type: Armor #same as regular sec armor
    modifiers:
      coefficients:
        Blunt: 0.70
        Slash: 0.70
        Piercing: 0.70
        Heat: 0.80
 ```
 нуарный плащ детектива
 ```yaml
     - type: Armor
      modifiers:
        coefficients:
          Blunt: 0.8
          Slash: 0.8
          Piercing: 0.6
          Heat: 0.9
```
ClothingOuterCoatWarden
```yaml
  - type: Armor
    modifiers:
      coefficients:
        Blunt: 0.7
        Slash: 0.7
        Piercing: 0.7
        Heat: 0.7
```
альтернатива:
```yaml
    - type: Armor
      modifiers:
        coefficients:
          Blunt: 0.8
          Slash: 0.8
          Piercing: 0.4
          Heat: 0.8
```
 я только защиту от взрывов не убрал у хоса и вардена, что сейчас пофикшу
 не я делал эти айтемы
 и не я писал их броню
 компонент защиты **должен** перезаписываться
 можешь сам сделать пр, чтоб, наши вещи были точно такими же как их предшественники

---

остальное ладно, но гравированая зажигалка должна быть одна, и ты унаследываешь плащ где она есть и у нас их выходит две, потому унаследуй из лодаутов, чтобы не было две

---

> остальное ладно, но гравированая зажигалка должна быть одна, и ты унаследываешь плащ где она есть и у нас их выходит две, потому унаследуй из лодаутов, чтобы не было две

ААААААААААА
пиздец
надо спать больше
понял :pray: